R/lsmeans.R

```diff
@@ -15,22 +15,22 @@
 #' to the mean.
 #'
 #' A final lsmean value is calculating by averaging these hypothetical
-#' patients. If `.weights` equals "proportional" then the values are weighted
+#' patients. If `.weights` equals `"proportional"` then the values are weighted
 #' by the frequency in which they occour in the full dataset. If `.weights`
-#' equals "equal" then each hypothetical patient is given an equal weight
+#' equals `"equal"` then each hypothetical patient is given an equal weight
 #' regardless of what actually occours in the dataset.
 #'
 #' Use the `...` argument to fix specific variables to specific values.
 #'
 #' See the references for identical implementations as done in SAS and
-#' in R via the emmeans package. This function attempts to re-implement the
-#' emmeans derivation for standard linear models but without having to include
-#' all of it's dependencies
+#' in R via the `emmeans` package. This function attempts to re-implement the
```

Comment:
Does this need to be in a code block as its the package name as opposed to a function / R object

Reply:
I don't know. I thought packages name should be in code block.. also rbmi when mentioned was in code block (in the joss paper for example).. I have not strong preference...